templates/Web/_composition/Flask/Page.Flask.AddSampleData.ForAllExceptList/server/sampleData$wts.Page_gpostaction.py

```diff
@@ -2,19 +2,19 @@
 import datetime
```

Comment:
Just wondering if the datetime causes problems. Since theres no differentiator between this and the shortLoremIpsum. When you generate grid and list for example only 1 short lorem ipsum is generated right?

Reply:
That's a good catch. It does combine the macros. But currently, I am not able to think of a way to separate that. I mean we need to have some kind of default text between macros to avoid combining.
Hmm, need to think about it

Reply:
Actually, I can fix it. We need to swap the `short_lorem_ipsum` and `long_lorem_ipsum` since `long_lorem_ipsum` is just used for grid/masterDetail sample data. This is not a generalized good solution but in consideration to the constraints in CoreTS, this is the best we can do.
I will make a small PR for that